ext/dom/php_dom.stub.php

```diff
@@ -323,6 +323,9 @@ class DOMNode
     /** @readonly */
     public ?DOMNamedNodeMap $attributes;
 
+    /** @readonly */
+    public bool $isConnected;
+
```

Comment:
As this is a new property, I *think* you can just use the normal keyword. @kocsismate is my assumption correct?

Reply:
Yes, absolutely, if the implementation is in line with `readonly`! :)

Reply:
It looks like this is more like a calculated field than a real readonly property (the value can change multiple times even though it's not possible to write by users), so the `@readonly` is correct here!

Reply:
Thanks Máté, for clarifying!